File: python/ejer_47.py
# 47. Buscar un número y mostrar su posición
arr = [10, 23, 45, 67, 89, 45, 12]
target = 45
positions = [i for i, x in enumerate(arr) if x == target]
print('Arreglo:', arr)
print(f'Buscando: {target}')
if positions:
    print(f'Encontrado en posición(es): {positions}')
else:
    print('No encontrado')

File: js/ejer_47.js
// 47. Buscar un número y mostrar su posición
const arr = [10, 23, 45, 67, 89, 45, 12];
const target = 45;
const positions = arr.map((x, i) => x === target ? i : -1).filter(i => i !== -1);
console.log('Arreglo:', arr);
console.log(`Buscando: ${target}`);
if (positions.length > 0) {
  console.log(`Encontrado en posición(es): ${positions}`);
} else {
  console.log('No encontrado');
}
